README.md

```diff
@@ -23,18 +23,20 @@ DD_SERVICE=<The name of the service you want to report>
 DD_SITE=<The Datadog site to upload results to>
 ```
 
-Depending on your CI service, you must also set the environment variables to be read from the test executions. See [CI Providers Environment Variables](https://docs.datadoghq.com/continuous_integration/setup_tests/swift/?tab=swiftpackagemanager#ci-provider-environment-variables) `EnvironmentKeys.swift` for details of your specific CI.
+Depending on your CI service, you must also set the environment variables to be read from the test executions. See [CI Providers Environment Variables](https://docs.datadoghq.com/tests/setup/swift/#ci-provider-environment-variables) for details of your specific CI.
 
-## UITests
+## UI Tests
 
-For UITests, both the test target and the application running from the UITests must link with the framework, environment variables only need to be set in the test target, since the framework automatically injects these values to the application.
+For UI Tests SDK will automatically integrate with the RUM SDK in your application. Environment variables only need to be set in the test target, since the framework automatically injects these values to the application.
```

Comment:
```suggestion
For UI Tests, this SDK will automatically integrate with the RUM SDK in your application. Environment variables only need to be set in the test target, since the framework automatically injects these values to the application.
```